src/sources/lolalytics.js

```diff
@@ -0,0 +1,304 @@
+    }
+
+    const filtered_items = R.uniqBy(R.prop('id'), items); //  filter out duplicates, mainly just the second health pot from converting buscuits
+
+    let block = {
+      type: title,
+      items: filtered_items
+    };
+
+    return block;
+  }
+
+
+  //  Iterate the sections
+  for (let i = 0; i < build_sections_to_convert.length; i++) {
+    //  Iterate the columns in that section
+    const current_section = build_sections_to_convert[i];
+
+    if (!split_sort) {
+      let block = convertBuildSectionJsonToBlock(R.reverse(sortByWinRate(current_section.json)), `${T.t('highest_win', true)} ${T.t(current_section.title.split(' ').join('_'))}`);
+      blocks.push(block);
+      block = convertBuildSectionJsonToBlock(R.reverse(sortByPickRate(current_section.json)), `${T.t('most_freq', true)} ${T.t(current_section.title.split(' ').join('_'))}`);
+      blocks.push(block);
+    } else if (R.toLower(split_sort) === 'win') {
+      let block = convertBuildSectionJsonToBlock(R.reverse(sortByWinRate(current_section.json)), `${T.t(current_section.title.split(' ').join('_'))}`);
+      blocks.push(block);
+    } else {
+      let block = convertBuildSectionJsonToBlock(R.reverse(sortByPickRate(current_section.json)), `${T.t(current_section.title.split(' ').join('_'))}`);
+      blocks.push(block);
+    }
+  }
+
+  let riot_json = R.merge(R.clone(default_schema, true), {
+    champion: champion_name,
+    title: `Lolalytics ${champion_name} ${T.t(R.toLower(position), true)} ${store.get('lolalytics_ver')}`,
+    blocks: trinksCon(blocks, skills)
+  });
+
+  if (split_sort) {
+    if (R.toLower(split_sort) === 'win') {
+      riot_json.title += ' ' + T.t('highest_win', true);
+    } else {
+      riot_json.title += ' ' + T.t('most_freq', true);
+    }
+  }
+
+  if (store.get('settings').locksr) riot_json.map = 'SR';
+
+  return createChampionifyJson(riot_json, position, champion_name, split_sort);
+}
+
+function parseSkillsData(cheerio_data) {
+  let $c = cheerio_data;
+  let skill_sections = [];
+  $c('td.summarytitle').each(function(i, element) {
+    if ($c(this).text() === 'Abilities') {
+      let innerSkillsTable = $c(this).parent().find('table.summaryskillstable');
+      let rows = $c(innerSkillsTable).find('tr');
+
+      $c(rows).each(function(j, row) {
+        if (j === 0)
+          return; //  Skip the header row
+
+        let skill_order = $c(row).find('td.summaryskills > div.summaryspellkey').text().split('').join('.');
+        let win_rate = parseFloat($c(row).find('td.win').text());
+        let pick_rate = parseFloat($c(row).find('td.popularity').text());
+
+        let skill_section = {
+          skills: skill_order,
+          win: win_rate,
+          pick: pick_rate
+        };
+        skill_sections.push(skill_section);
+      }); //  end each row
+    } //  end if abilities section
+  }); //  end each td.summarytitle
+
+  var highest_win_func = R.sortBy(R.prop('win'));
+  var most_freq_func = R.sortBy(R.prop('pick'));
```

Comment:
You use these two in your `createRiotJson` function. If you move them in to global space it prevents you from having to reassign them. :)